src/main/java/com/example/bookshop/repository/BookRepository.java

```diff
@@ -2,6 +2,8 @@
 
 import com.example.bookshop.model.Book;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
+@Repository
```

Comment:
Check what will be if you erase this annotation

Reply:
Nothing happens becouse spring auto recognises repositories that extend JpaRepo